### DISCUSSION TITLE
Icons on càmera view showing Is detect, Redford, snapshot doesnt correspond with real settings

### DISCUSSION THREAD

**Discussion by divemasterjm** [blakeblackshear/frigate#14456] [General]
On last build 0.15 , icons are not showing real status of recording, snapshot and detect, if you change anything on ha or even with frigate add-on log is reflecting the change but icons doesn't, if I activate them on webui remains but if I send Matt message to change it or with frigate add-on doesn't change

**Comment by hawkeye217** [comment]
Thanks, this is fixed in https://github.com/blakeblackshear/frigate/pull/14466

**Comment by divemasterjm** [comment]
Still some issues with detection icon
